lib/config.go

```diff
@@ -66,12 +66,16 @@ func sourceProfile(p string, from Profiles) string {
 	return p
 }
 
-func (p Profiles) GetValue(profile string, config_key string) (string, string, error) {
+func (p Profiles) GetValue(profile string, config_key string, recursive bool) (string, string, error) {
```

Comment:
I think I'd prefer a new `GetValueRecursive` function 

https://github.com/segmentio/aws-okta/wiki/Style-Guide#api-compatibility

Reply:
I started to make this change, but ended up creating a `GetDirectValue` function instead. Here's why:

I was calling the existing (before this PR) `GetValue` behavior recursive. Upon revisiting this, it occurred to me that it was a bit misleading because it doesn't descend past the immediate `source_profile`.

To address the underlying issue, I needed a function that only looked at settings set directly on the given profile, which is what `GetDirectValue` does. My most recent commit restores the original `GetValue` behavior, but I did make it use `GetDirectValue` under-the-hood to DRY things up a little. Happy to restore the old implementation if you'd prefer less indirection over DRY here.